Babylon/groups/azure/groups/adx/commands/run_script.py

```diff
@@ -36,11 +36,11 @@ def run_script(kmc: KustoManagementClient, cluster_name: str, resource_group_nam
     script_name = f"{int(time.time() // 1)}-{script_file.name}"
     with open(script_file) as _script_file:
         logger.info(f"Reading {script_file}")
-        script_content = _script_file.read().replace("<database name>", database_name)
+        script_content = _script_file.read().replace("<database name>", adx_database_name)
         logger.info("Sending script to database.")
         s = kmc.scripts.begin_create_or_update(resource_group_name=resource_group_name,
-                                               cluster_name=cluster_name,
-                                               database_name=database_name,
+                                               adx_cluster_name=adx_cluster_name,
+                                               adx_database_name=adx_database_name,
```

Comment:
```suggestion
                                               cluster_name=adx_cluster_name,
                                               database_name=adx_database_name,
```
It's probably a mistake ?